config/appsignal.yml

```diff
@@ -7,14 +7,25 @@ default:
   working_directory_path: '/tmp/appsignal'
 
   # Actions that should not be monitored by AppSignal
-  # ignore_actions:
-  #   - ApplicationController#isup
+ignore_actions:
+  - 'LinkCrawlWorker#perform'
+  - 'ActivityPub::DeliveryWorker#perform'
+  - 'VerifyAccountLinksWorker#perform'
+  - 'ActivityPub::SynchronizeFeaturedCollectionWorker#perform'
+  - 'ActivityPub::FetchRepliesWorker#perform'
+  - 'ActivityPub::ProcessingWorker#perform'
+  - 'UpdateStatusStatWorker#perform'
+  - 'FeedInsertWorker#perform'
+  - 'ChannelFeedWorker#perform'
+  - 'ThreadResolveWorker#perform'
 
```

Comment:
cutting down on the noise in AppSignal. These are _mostly_ Mastodon specific errors